cloud-third-party-controller/views/customui.html

```diff
@@ -0,0 +1,526 @@
+        html = '';
+        n++;
+      }
+      if(n > 0) $("#profiles_loaded_chk").prop('checked', true); 
+    }
+
+    //Send the stream auth data to the device
+    function configStream(id) {
+      var stream = {};
+      stream["id"] = id;
+      stream["stream_key"] = $("#output_id_" + id + "_key").val();
+      stream["stream_url"] = $("#output_id_" + id + "_url").val();
+
+      $.ajax({
+        url: "/streams/" + id,
+        data: JSON.stringify(stream),
+        method: 'POST',
+        contentType: 'application/json',
+        processData: false,
+        success: function() {
+          streamReady = true;
+          //Change the stream save button to indicate it worked.
+          $('#output_stream_' + id + "_btn").removeClass("unsaved");
+          $('#output_stream_' + id + "_btn").addClass("stream-saved");
+          
+          //Change the big button to indicate it's ready to stream.
+          $("#go_live_button").removeClass("select-button-default");
+          $("#go_live_button").addClass("select-button-ready");
+          $("#stream_button_text").text("Ready!");
+        },
+        error: function(error) {
+          alert("Could not save stream data:\n\n" + JSON.stringify(error));
+        }
+      });
+    }
+
+    //The big red button
+    function goLive() {
+      if(!streamReady) {
+        alert("Please set up one or more output streams in order to go live.");
+        return false;
+      }
+
+      $.ajax({
+        url: "/golive/",
+        method: 'POST',
+        contentType: 'application/json',
+        processData: false,
+        success: function(data) {
+          // TODO: make this update based on the actual outputs statuses, rather than toggling and assuming all is working properly
+          if(isStreaming) {
+            if($("#go_live_button").hasClass("select-button-active")) {
+              $("#go_live_button").removeClass("select-button-active");
+              $("#go_live_button").addClass("select-button-ready");
+              $("#stream_button_text").text("Ready!");
+            }
+            isStreaming = false;
+          } else {
+            $("#go_live_button").removeClass("select-button-ready");
+            $("#go_live_button").addClass("select-button-active");
+            $("#stream_button_text").text("STREAMING\nLIVE");
+            isStreaming = true;
+          }
+        },
+        error: function(error) {
+          alert("An error occurred:\n\n" + JSON.stringify(error));
+        }
+      });
+    }
+
+    function submitPAT(token) {
+      var orgLabel = document.getElementById("org_label");
+      orgLabel.textContent = "Retrieving Organizations...";
+
+      $.ajax({
+        url: "/submitPAT/" + token,
+        method: 'POST',
+        contentType: 'application/json',
+        success: function(data) {
+          var orgSelect = document.getElementById("orgDropdown");
+          var orgs = JSON.parse(data)["orgs"];
+          for (var org in orgs) {
```

Comment:
It could be useful to add ordering by `org_name` because as it is, it's a bit messy for Videon admins who have all the orgs in the lit